pkg/version/version.go

```diff
@@ -21,21 +21,10 @@ var (
 	buildDate string
 )
 
-// Info contains versioning information.
-// TODO: Add []string of api versions supported? It's still unclear
-// how we'll want to distribute that information.
-type Info struct {
-	Major      string `json:"major"`
-	Minor      string `json:"minor"`
-	GitCommit  string `json:"gitCommit"`
-	GitVersion string `json:"gitVersion"`
```

Comment:
do we loose the GitVersion and BuildDate?

Reply:
We're taking the upstream `Info` struct, which has all those fields.